src/components/TodoItem.js

```diff
@@ -0,0 +1,56 @@
+import {Text, StyleSheet, TouchableOpacity, View, CheckBox} from 'react-native';
+import React from 'react';
+import {dateToDDMMYYY} from '../utils/utils';
+
+export const TodoItem = ({item}) => {
+  return (
+    <View style={styles.wrapper}>
+      <View style={styles.todo}>
+        <CheckBox style={styles.completed} value={item.completed} />
+        <View style={styles.text_container}>
+          <Text
+            style={
+              item.completed === true
```

Comment:
Nāo precisas do `=== true`
```suggestion
              item.completed
```